python/meterpreter/ext_server_stdapi.py

```diff
@@ -362,8 +359,8 @@ class WINHTTP_CURRENT_USER_IE_PROXY_CONFIG(ctypes.Structure):
 
     class LUID(ctypes.Structure):
         _fields_ = [
-            ('LowPart',  wintypes.DWORD),
-            ('HighPart', wintypes.LONG)
+            ('LowPart',  ctypes.c_uint32),
+            ('HighPart', ctypes.c_uint32)
```

Comment:
Just confirming if this is meant to be `c_ulong` 👀

Reply:
Actually, I guess it should be just `ctypes.c_long`. Might be important given that long is signed where as DWORD / c_uint32 is not. I'll get that fixed.